src/main.js

```diff
@@ -79,21 +80,3 @@ function update () {
   const htmlForPreview = createHtml({ html, js, css })
   $('iframe').setAttribute('srcdoc', htmlForPreview)
```

Comment:
I wanted to change it this way, but I wasn't sure if this could reduce legibility. 
what do you think?

```suggestion 
  $('iframe').setAttribute('srcdoc', createHtml({ html, js, css }))
```